app/controllers/statistics_controller.rb

```diff
@@ -12,6 +14,31 @@ def collection
     @all_indexers     = build_user_array(DeedType::PAGE_INDEXED)
   end
 
+  def export_csv
+    header = ['User Login', 'User Name', 'Email', 'Opt-In']
+    owner = User.find 'ushmmarchives'
+    collection_ids = owner.collections.map { |c| c.id }.sort
+    deed_map = Deed.where(:collection_id => collection_ids).group(:user_id, :collection_id).count
+    user_ids = deed_map.keys.map {|e| e[0]}.uniq
```

Comment:
Now get the list of unique users who worked on those collections by plucking out the first element of the key `[user_id, collection_id]` -- this is the `user_id`, and we want the unique user IDs.

This creates an array of user IDs for all users who worked on this owner's collections.